docs/source/getting-started.rst

```diff
@@ -198,6 +198,8 @@ Slack notifications
 Here we will configure a built-in Spidermon action that sends a pre-defined message to
 a Slack channel using a bot when a monitor fails.
 
+  .. warning:: Slack integration **requires** Python 3.6 or later.
```

Comment:
**suggestion (non-blocking)** As Python 3.5 support was dropped, there is no need to include this warning on the docs now.